crates/core/src/client_events/websocket.rs

```diff
@@ -325,7 +337,10 @@ async fn websocket_commands(
         }
     };
 
-    ws.on_upgrade(on_upgrade)
+    // Increase max message size to 100MB to handle contract uploads
+    // Default is ~64KB which is too small for WASM contracts
+    ws.max_message_size(100 * 1024 * 1024)
```

Comment:
I would consider this to be a temporal fix, we need to look into different ways of sending messages this large (maybe a regular multipart http request) cause this a bit of an antipattern for ws. I know is over local host but still, we can do better here .

I think eventually we want to stream that right into disk to not hold into such large things in memory for very long, but that requries a bigger architetural change.